packages/hierarchies-react/src/presentation-hierarchies-react/itwinui/TreeNodeRenderer.tsx

```diff
@@ -135,6 +136,7 @@ export const TreeNodeRenderer: ForwardRefExoticComponent<TreeNodeRendererProps &
     return (
       <Tree.Item
         {...treeItemProps}
+        className={"stateless-tree-node"}
```

Comment:
I don't think we should use the term "stateless tree" anymore - mind changing this to `presentation-hierarchies-node`?

Reply:
Changed